CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 1.2.0-rc1 (2020-11-06)
```

Comment:
Since we are releasing this Monday, we might as well use the date here : 2020-11-09